src/freertos_drivers/common/WifiDefs.cxx

```diff
@@ -31,6 +31,8 @@
  * @date 3 February 2017
  */
 
+#include "freertos_drivers/common/WifiDefs.hxx"
```

Comment:
Can this be reduced to just the hxx filename here and avoid paths?

Reply:
I would not recommend that. We can always guarantee to have the openmrn/src directory on our include path. Beyond that, I don't think we guarantee to add additional paths.